packages/pymedphys_deliverydata/src/pymedphys_deliverydata/dicom/conversion.py

```diff
@@ -29,62 +29,49 @@
 
 import numpy as np
 
-from .core import (
-    DeliveryData, find_relevant_control_points)
+from pymedphys_utilities.transforms import convert_angle_to_bipolar
+from pymedphys_utilities.algorithms import maintain_order_unique
 
+from pymedphys_dicom.rtplan import (
+    get_gantry_angles_from_dicom,
+    build_control_points,
+    replace_fraction_group,
+    replace_beam_sequence,
+    restore_trailing_zeros)
```

Comment:
I've decided to follow suite with your convention of keeping packages each on a new line. That means changing a package only is clearer with version control.